charms/jupyter-ui/config.yaml

```diff
@@ -19,3 +19,22 @@ options:
     type: boolean
     default: false
     description: Whether cookies should require HTTPS
+  jupyter-images:
+    type: string
+    default: |
```

Comment:
Should default images be the same as in `track/1.7`?
```
    - charmedkubeflow/jupyter-scipy:v1.7.0_20.04_1
    - charmedkubeflow/jupyter-pytorch-full:v1.7.0_20.04_1
    - charmedkubeflow/jupyter-pytorch-cuda-full:v1.7.0_20.04_1
    - charmedkubeflow/jupyter-tensorflow-full:v1.7.0_20.04_1
    - charmedkubeflow/jupyter-tensorflow-cuda-full:v1.7.0_20.04_1
```
As I see it those are ROCKs that were integrated into 1.7.0 recently: https://github.com/canonical/notebook-operators/pull/303